src/main/java/characterpicker/infrastructure/CompletionWeightCalculator.java

```diff
@@ -0,0 +1,56 @@
+package characterpicker.infrastructure;
+
+import java.util.List;
+
+public class CompletionWeightCalculator {
+
+    private CompletionWeightCalculator(){
+        throw new IllegalStateException("Utility class");
+    }
+    //TO DO
+    //change List name
+     static Integer calculateCompletionWeight(List<String> list) {
+
+        double sumOfWeights = 0;
+
+        for (String boss : list) {
+            if (boss.contains("mom's hearth")) {
+                sumOfWeights += 0.1;
+            }
+            if (boss.contains("isaac")) {
+                sumOfWeights += 0.1;
+            }
+            if(boss.contains("???")){
+                sumOfWeights+=0.1;
+            }
+            if(boss.contains("satan")){
+                sumOfWeights+=0.1;
+            }
+            if(boss.contains("the lamb")){
+                sumOfWeights+=0.1;
+            }
+            if(boss.contains("boss rush")){
+                sumOfWeights+=0.1;
+            }
+            if(boss.contains("ultra greed")){
+                sumOfWeights+=0.1;
+            }
+            if(boss.contains("hush")){
+                sumOfWeights+=0.25;
+            }
+            if(boss.contains("mega satan")){
+                sumOfWeights+=0.25;
+            }
+            if (boss.contains("delirium")) {
+                sumOfWeights += 0.5;
+            }
+            if (boss.contains("mother")) {
+                sumOfWeights += 0.5;
+            }
+            if (boss.contains("the beast")) {
+                sumOfWeights += 0.5;
+            }
+        }
+        return (int) Math.round(sumOfWeights);
```

Comment:
Pasowałoby pomyśleć, aby w przyszłości weight był liczbą zmiennoprzecinkową, bo tutaj ucinamy dane. Jak ktoś poda np. sam boss rush to nie ma to teraz w ogóle wpływu na wagę, bo zaokrągli w dół.